tcf_website/management/commands/index_elasticsearch.py

```diff
@@ -1,6 +1,4 @@
-"""
-Modules used
-"""
+"""Modules used"""
 import os
 import json
 import requests
```

Comment:
do we need to add this to requirements.txt?

Reply:
no because it is a built-in python library

Official documentation:
https://requests.readthedocs.io/en/master/

Reply:
It's not built in: https://pypi.org/project/requests/

Reply:
fixed